### DISCUSSION TITLE
Way to do planned cabling that overlaps with existing cabling?

### DISCUSSION THREAD

**Discussion by nmiller2600** [netbox-community/netbox#9038] [Q&A]
I'm trying to model out a move of some equipment and provide a cut-sheet to my datacenter team so that they can perform the cabling now, and then we can cut over the the new connections on the night of the change.

In my case, I'm using unique Network Interfaces (different switches, different ports) but the same interfaces on the server end.

Is it possible to make it so that :
cable types of Connected and planned only overlap within themselves, rather than with each other
or:
Planned cabling can overlap with anything, possibly with warning
or:
 is there a better way to accomplish what i'm trying to do?

I'd like to have the currently intact cabling as CONNECTED
map out a new 'planned' cable betwen a device and another.
Then at migration time, CONNECTED would transfer to "Decomissioned" and the planned would transition to connected.

I've been using Netbox to enforce my single-port-single-connection which has been absolutely critical to my needs, but having a planned cable swing feels like a valid use case as well.  

Any workarounds/thoughts appreciated.

**Comment by bluikko** [comment]
I have been toying around with a "change request" feature. It could work for example something like this:
1. Click "Begin change request" button. Perhaps fill in a form for details of the change request.
2. Do changes in NetBox. The changes would be tracked somehow but actual data in NetBox would not change. Perhaps the changes could be grouped in an item in the "Change requests" area. Maybe the relevant objects would have an icon to signify pending change that is linked to the change request.
3. Click "Finish change request" button.
4. The change request would be marked as "Pending" in the "Change requests" area.
5. Print/mail/whatever that particular change request or a link to it to whoever would make the change. Or notify in mail/webhook automatically if the change was assigned to a NetBox user.
6. Change is implemented.
7. The change request can be marked as "Completed" and NetBox data would update to reflect the changes.

The "change request" would be basically a step by step or object by object list of changes to do that could be given to level 1 staff.

I have no idea how feasible it would be to do something like this: I do not know how NetBox work internally. Because of that I've never thought out this in detail since maybe it's completely not a possibility in which case the time would be a loss...
